lua/mgn/cl_voxlist.lua

```diff
@@ -585,7 +585,7 @@ mgn.VOXList = {
 	["violated"] = {Path = "sound/vox/violated.wav", Duration = 0.895873},
 	["violation"] = {Path = "sound/vox/violation.wav", Duration = 0.937324},
 	["voltage"] = {Path = "sound/vox/voltage.wav", Duration = 0.708027},
-	["vox_login"] = {Path = "sound/vox/vox_login.wav", Duration = 10.288889},
+	-- ["vox_login"] = {Path = "sound/vox/vox_login.wav", Duration = 10.288889},
```

Comment:
If you're gonna remove it, remove it completely. There's no point in commenting it.